### ISSUE TITLE
Blank instead of "not selected" in task details view

### ISSUE THREAD

**Issue by linuxcaffe** [CCExtractor/taskwarrior-flutter#375]
### Describe your improvement idea or feature request

In a quick scan of the details of a task, a blank field is for more obvious that a value has not been selected, than the current "not selected" text

### How would you like to see it implemented?

UI back end..?

### Would you like to work on this issue?

Yes

### By submitting this issue, I have confirmed that:

- [X] I have starred the repo ⭐ and watched 👀 it on GitHub and followed the contribution guidelines.

**Comment by linuxcaffe** [comment]
Ok @BrawlerXull , gimme a clue where I would find that code!

**Comment by BrawlerXull** [comment]
U may find it in `lib/app/modules/detailRoute/views/` but I think Not selected looks fine @Pavel401 what's your take on it

**Comment by linuxcaffe** [comment]
@BrawlerXull I agree, **Not selected** looks fine, but my argument against it is one of "visual noise", that is anything that is not "signal", or in this case, text on a list of attributes, meaning literally "nothing". As someone who has waded through some big task lists, I have found that reducing the display of non-information helps me scan for relevant details more effectively. Don't get me wrong, I think you guys are doing amazing good work, and I hope you won't get your collective noses out of joint if I point out a few more places where less-is-more. Thanks!

**Comment by arjav1528** [comment]
@linuxcaffe  can you assign me this issue

**Comment by linuxcaffe** [comment]
Oh you know I would, but I ain't got the auth!

**Comment by arjav1528** [comment]
@Pavel401 can you assign me this issue??

**Comment by asmitjain71** [comment]
@Pavel401 Please assign this issue to me. I want to work on this.

**Comment by Pavel401** [comment]
@arjav1528 what's the update?

**Comment by asmitjain71** [comment]
> [@Pavel401](https://github.com/Pavel401) Please assign this issue to me. I want to work on this.

Hi please Let me Know

**Comment by Anjalirajaramker** [comment]
is this issue still available??

**Comment by sandy4242** [comment]
@linuxcaffe is it available?? 
@arjav1528 are you working midway??
